### ISSUE TITLE
Communicating between asterisk 1.8 and asterisk 16 servers

### ISSUE THREAD

**Issue by dannydiazvonage** [asterisk-java/asterisk-java#401]
I apologize in advance if this is the wrong place to ask but couldn't seem to find confirmation of this online.  I was trying to determine if asterisk-java 3.x library would actually be compatible in communicating with asterisk 1.8.  Kind of have a situation where our Java application would have to talk with both asterisk version 1.8 and asterisk 16 until a full upgrade to asterisk 16 would be complete.  Was under the impression that 3.x was not compatible with asterisk 1.8 and was trying to confirm if that's actually true or if asterisk-java 3.x can indeed speak with both asterisk 1.8 and asterisk 16 versions.

**Comment by bsutton** [comment]
From recollection, there are some difference but most of the API should
work.

So it will depend on which bits of the API you are doing to use.

Sorry not much help.

**Comment by dannydiazvonage** [comment]
That actually is a little helpful.  We have a pretty extensive regression test CI of a lot of common call flows so the ones that break are probably those API boundaries that may be different.  As it stands it seems like 3.x currently passes the same call flows as 2.x is doing so we probably aren't using those API bits.  I was expecting 3.x to completely break most of it since I assumed it was a different implementation and when I saw them passing I was scratching my head as I thought it only worked with asterisk 10-18.  Maybe we lucked out.
